subprojects/docs/src/docs/userguide/javaPlugin.xml

```diff
@@ -396,7 +399,10 @@
                 <td>runtime</td>
                 <td>compile</td>
                 <td>-</td>
-                <td>Runtime dependencies</td>
+                <td>Runtime dependencies. This configuration is weakly deprecated in favor of the <code>implementation</code>, <code>runtimeElements</code>
```

Comment:
❌ `runtimeOnly` is the replacement for `runtime`, not `implementation`. `implementation is visible at compile time and is one of the replacements for `compile` (the other being `api` in the `java-library` plugin.